### PR TITLE
solved task

### PR DESCRIPTION
**Pull request by hulchak**
https://hulchak.github.io/react_goods-selector-js/

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -14,70 +15,73 @@ export const goods = [
...
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+export const App = () => {
+  const defaultSelectedGood = 'Jam';
```
Comment (etojeDenys) [inline src/App.jsx:44]: you can move it out of the component as constant

File: src/App.jsx [hunk 2]
```diff
@@ -14,70 +15,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const GoodRow = ({ good, selectedGood, onClick }) => {
+  const isSelected = selectedGood === good;
+  const buttonClassName = `button ${isSelected ? 'is-info' : ''}`;
+  const rowClassName = isSelected ? 'has-background-success-light' : '';
```
Comment (etojeDenys) [inline src/App.jsx:21]: better to use classnames package for such cases

File: src/App.jsx [hunk 3]
```diff
@@ -14,70 +15,73 @@ export const goods = [
...
+            type="button"
+            className="delete ml-3"
+            onClick={() => setSelectedGood(null)}
+          />
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      )}
```
Comment (etojeDenys) [inline src/App.jsx:71]: you can add condition only to text inside the `h1` tag

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Review by lerastarynets — APPROVED** [review]
lgtm 🚀

**Pull request closed without merging**